renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -829,7 +844,6 @@ SERIALISE_VK_HANDLES();
                                                                                                        \
   /* VK_AMD_display_native_hdr */                                                                      \
   PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_DISPLAY_NATIVE_HDR_SURFACE_CAPABILITIES_AMD)                     \
```

Comment:
A supported extension should not leave some elements as `PNEXT_UNSUPPORTED` still.